Repository: plone.formwidget.namedfile


Branch: refs/heads/master
Date: 2021-06-23T16:45:15+02:00
Author: Jens W. Klein (jensens) <jk@kleinundpartner.at>
Commit: https://github.com/plone/plone.formwidget.namedfile/commit/88270f7881e17adb86d702a6e5f18cf613811b50

pillow 9 exception is different

Files changed:
A news/50.bugfix
M plone/formwidget/namedfile/widget.rst

b'diff --git a/news/50.bugfix b/news/50.bugfix\nnew file mode 100644\nindex 0000000..d8aac5a\n--- /dev/null\n+++ b/news/50.bugfix\n@@ -0,0 +1 @@\n+Support for Pillow 8 in tests. [jensens]\ndiff --git a/plone/formwidget/namedfile/widget.rst b/plone/formwidget/namedfile/widget.rst\nindex a8fa0f2..0eb3060 100644\n--- a/plone/formwidget/namedfile/widget.rst\n+++ b/plone/formwidget/namedfile/widget.rst\n@@ -752,10 +752,11 @@ Check that we have a good image that PIL can handle::\n \n Note that PIL cannot open this bytes image, so we cannot scale it::\n \n-  >>> PIL.Image.open(six.BytesIO(content.image_field))\n-  Traceback (most recent call last):\n-  ...\n-  OSError: cannot identify image file...\n+  >>> try:\n+  ...     PIL.Image.open(six.BytesIO(content.image_field))\n+  ... except Exception as e:\n+  ...     print(e)\n+  cannot identify image file...\n \n Prepare for a new request cycle::\n \n'

Repository: plone.formwidget.namedfile


Branch: refs/heads/master
Date: 2021-06-23T21:04:03+02:00
Author: Maurits van Rees (mauritsvanrees) <m.van.rees@zestsoftware.nl>
Commit: https://github.com/plone/plone.formwidget.namedfile/commit/fa035fdf4bad4ae26cce7d8e73f6c58a24b9be97

Merge pull request #50 from plone/pillow-8

pillow 8 exception is different

Files changed:
A news/50.bugfix
M plone/formwidget/namedfile/widget.rst

b'diff --git a/news/50.bugfix b/news/50.bugfix\nnew file mode 100644\nindex 0000000..d8aac5a\n--- /dev/null\n+++ b/news/50.bugfix\n@@ -0,0 +1 @@\n+Support for Pillow 8 in tests. [jensens]\ndiff --git a/plone/formwidget/namedfile/widget.rst b/plone/formwidget/namedfile/widget.rst\nindex a8fa0f2..0eb3060 100644\n--- a/plone/formwidget/namedfile/widget.rst\n+++ b/plone/formwidget/namedfile/widget.rst\n@@ -752,10 +752,11 @@ Check that we have a good image that PIL can handle::\n \n Note that PIL cannot open this bytes image, so we cannot scale it::\n \n-  >>> PIL.Image.open(six.BytesIO(content.image_field))\n-  Traceback (most recent call last):\n-  ...\n-  OSError: cannot identify image file...\n+  >>> try:\n+  ...     PIL.Image.open(six.BytesIO(content.image_field))\n+  ... except Exception as e:\n+  ...     print(e)\n+  cannot identify image file...\n \n Prepare for a new request cycle::\n \n'

